java/bogdana_maksiuta/src/com/zoolatech/lecture3/tasks/_1/Drum.java

```diff
@@ -0,0 +1,16 @@
+package com.zoolatech.lecture3.tasks._1;
+
+public class Drum extends MusicalInstrument {
+    public Drum(String name, String sound) {
+        super(name, sound);
+    }
+
+    void play() {
+        super.play();
```

Comment:
You don't need to call the parent's method or even define the method `play` in the children classes

Reply:
fixed